modules/retail/pages/retail-offline.adoc

```diff
@@ -0,0 +1,33 @@
+[[retail.offline]]
+= Offline Use
+
+If the {susemgr} Server is offline, you can still perform a reboot on the terminals.
+If the connection between the branch server and the terminals is lost or unstable, you can still perform an update on the terminals, using caching.
+
+[IMPORTANT]
+====
+Offline features are available only for {susemgr} Server 4.0 and above.
+You must have the Saltboot and PXE formula packages installed.
+====
+
+== Offline Terminal Reboot
+
+If the {susemgr} Server is offline, and a terminal is rebooted, it will fall back to a previously installed image.
+
+This will occur in these situations:
+* If the saltboot.py module on the {susemgr} Server has not been synchronized within a specified time (default value is 60 seconds).
+This is the first step in the usual salt boot procedure, so if it fails, it indicates that there is a problem with the {susemgr} Server.
+* If the {susemgr} Server does not wait for the salt key to be accepted by the terminal.
+* If the root partition is specified on the kernel command line (handled by the PXE formula), is mountable (and is not encrypted), and contains [path]``/etc/ImageVersion`` (which is created by Kiwi).
+
+
+
+== Offline Terminal Updates
+
+If there is limited bandwidth between the branch server and the terminal, for example if the branch server is not in the same location as the terminal, then images can be cached in advance, and the upgrade performed on the terminals later on, without a network connection.
```

Comment:
sentence too long (I know this is in draft state, but now because I started with reviewing I must mention all my observations).

Reply:
As I said above, a low bandwidth connection is needed during reboot.